airflow/api_fastapi/core_api/routes/public/__init__.py

```diff
@@ -68,10 +68,10 @@
 authenticated_router.include_router(plugins_router)
 authenticated_router.include_router(pools_router)
 authenticated_router.include_router(providers_router)
+authenticated_router.include_router(xcom_router)
 authenticated_router.include_router(task_instances_router)
 authenticated_router.include_router(tasks_router)
 authenticated_router.include_router(variables_router)
-authenticated_router.include_router(xcom_router)
```

Comment:
nit: undo this change.
We are maintaining it in an alphabetical order

Reply:
This was required due to a URL conflict with patch_task_instance endpoint:
- patch_task_instance: `"/{task_id}/{map_index}"`
- getEntries: `/{task_id}/getEntries`

Therefore it would throw a 422 error (because it tries to use "getEntries" as map_index which is defined as an int).

By moving xcom before task_instances, the xcom url is attempted first which resolves the issue, while the task instance endpoint is not affected.

Reply:
Great thanks for the explanation.